src/components/pip/pip-child.tsx

```diff
@@ -87,7 +90,7 @@ export class PipChild extends Component<PIPChildComponentProps> {
         <Button
           className={styles.iconContainer}
           onClick={onInversePIP}
-          tooltip={{label: this.props.switchScreen!, type: 'bottom-left'}}
+          tooltip={{label: this.props.layout === Layout.PIP ? this.props.switchToSecondaryScreen! : this.props.switchToPrimaryScreen!, type: 'bottom-left'}}
```

Comment:
same here
I would recommend move the condition inside translates.
Example: https://github.com/kaltura/playkit-js-navigation/blob/master/src/components/navigation/plugin-button/index.tsx#L10-L14